test/index.spec.ts

```diff
@@ -47,29 +48,34 @@ function nodeCryptoHashInto(input: Buffer, output: Buffer): void {
   }
 }
 
-describe("should hash similarly to crypto.createHash('sha256')", () => {
+it("should hash similarly to crypto.createHash('sha256')", () => {
   for (let i = 1; i <= 16; i++) {
-    test(`No of Chunks=${i}`, () => {
-      for (let j = 0; j < 255; j++) {
-        const input = Buffer.alloc(CHUNK_SIZE * i, j);
-        expectEqualHex(hash(input), nodeCryptoHash(input));
-      }
-    });
+    for (let j = 0; j < 255; j++) {
+      const input = Buffer.alloc(CHUNK_SIZE * i, j);
+      expectEqualHex(hash(input), nodeCryptoHash(input));
+    }
   }
 });
 
-describe("should hashInto similarly to crypto.createHash('sha256')", () => {
+it("should asyncHash similarly to crypto.createHash('sha256')", async () => {
   for (let i = 1; i <= 16; i++) {
-    test(`No of Chunks=${i}`, () => {
-      for (let j = 0; j < 255; j++) {
-        const input = Buffer.alloc(CHUNK_SIZE * i, j);
-        const output1 = Buffer.alloc((CHUNK_SIZE / 2) * i);
-        const output2 = Buffer.alloc((CHUNK_SIZE / 2) * i);
+    for (let j = 0; j < 255; j++) {
+      const input = Buffer.alloc(CHUNK_SIZE * i, j);
+      expectEqualHex(await hashAsync(input), nodeCryptoHash(input));
+    }
+  }
+});
```

Comment:
Just split this into a separate test so there is only one expect per test.  Also cleaned up the output, i added a describe on PR#3 to show the chunk size but seems unnecessary in retrospect